src/components/header/__snapshots__/header.test.tsx.snap

```diff
@@ -42,12 +42,20 @@ exports[`OuiHeader sections render breadcrumbs and props 1`] = `
     aria-label="breadcrumb"
     class="ouiBreadcrumbs ouiHeaderBreadcrumbs ouiBreadcrumbs--truncate"
   >
-    <span
-      aria-current="page"
-      class="ouiBreadcrumb ouiBreadcrumb--last"
+    <div
+      class="ouiBreadcrumbWall"
     >
-      Breadcrumb
-    </span>
+      <div
+        class="ouiBreadcrumbWrapper ouiBreadcrumbWrapper--first ouiBreadcrumbWrapper--last"
+      >
+        <span
+          aria-current="page"
+          class="ouiBreadcrumb ouiBreadcrumb--last"
```

Comment:
Is it necessary to have a `--last` class on both the wrapper and the breadcrumb? It's OK, just a little redundant and therefore fragile.

Reply:
https://github.com/opensearch-project/oui/issues/377